boards/arm/ene_kb1200_evb/support/eneispif_support.patch

```diff
@@ -0,0 +1,495 @@
+diff --git a/src/flash/nor/Makefile.am b/src/flash/nor/Makefile.am
```

Comment:
-1 to store patches like this in tree, please, make this available (with instructions) in a vendor site/repository

Reply:
@ene-steven instead you can submit this patch to our fork of openocd:
https://github.com/zephyrproject-rtos/openocd

Reply:
[https://github.com/zephyrproject-rtos/openocd/pull/60](url)